From 323cfd86e886db5fb25bb76294946c36fbf15cd8 Mon Sep 17 00:00:00 2001
From: AdrianDC <radian.dc@gmail.com>
Date: Thu, 19 Nov 2015 21:25:34 +0100
Subject: [PATCH 7/8] NFC: Clean duplicated and unknown permissions

 * Unknown permission android.permission.NFC_UNLOCK in package com.android.nfc
 * CHANGE_WIFI_STATE and ACCESS_WIFI_STATE duplicated

Change-Id: Ic785e41657a7c6eb7d97c04957d422c5471bc916
Signed-off-by: AdrianDC <radian.dc@gmail.com>
---
 AndroidManifest.xml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 6009074c..0c2bb64a 100755
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -22,7 +22,6 @@
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
     <uses-permission android:name="android.permission.BLUETOOTH_PRIVILEGED" />
     <uses-permission android:name="android.permission.NFC" />
-    <uses-permission android:name="android.permission.NFC_UNLOCK" />
     <uses-permission android:name="android.permission.BIND_NFC_SERVICE" />
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.STATUS_BAR" />
@@ -44,8 +43,6 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.MANAGE_USERS" />
     <uses-permission android:name="android.permission.INTERNAL_SYSTEM_WINDOW" />
-    <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
-    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.SET_ACTIVITY_WATCHER" />
     <uses-permission android:name="android.permission.CONNECTIVITY_INTERNAL" />
     <uses-permission android:name="android.permission.NFC_HANDOVER_STATUS" />
-- 
2.17.1

